### PR TITLE
fio: Add version 3.18

### PR DESCRIPTION
**Pull request by AntonOks**

### CODE REVIEW

File: bucket/fio.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "3.18",
+    "description": "Flexible I/O Tester",
+    "homepage": "https://bsdio.com/fio/",
+    "license": {
+        "identifier": "GPL-2.0-only",
+        "url": "https://github.com/axboe/fio/blob/master/COPYING"
+    },
```
Comment (Ash258) [inline bucket/fio.json]:
```suggestion
    "license": "GPL-2.0-only",
```

File: bucket/fio.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://bsdio.com/fio/releases/fio-3.18-x64.zip",
+            "hash": "0B062AE1BCFFE0204ACF9DFCC3D88AD55309A205C196781CD95C04248F064241",
```
Comment (Ash258) [inline bucket/fio.json]:
```suggestion
            "hash": "0b062ae1bcffe0204acf9dfcc3d88ad55309a205c196781cd95c04248f064241",
```

File: bucket/fio.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            "extract_dir": "fio-3.18-x64"
+        },
+        "32bit": {
+            "url": "https://bsdio.com/fio/releases/fio-3.18-x86.zip",
+            "hash": "BAB07B3F084EBE708F3E3D3032CA686BA5B47E5EC82E6BFC8990F27E47031648",
```
Comment (Ash258) [inline bucket/fio.json]:
```suggestion
            "hash": "bab07b3f084ebe708f3e3d3032ca686ba5b47e5ec82e6bfc8990f27e47031648",
```

File: bucket/fio.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            "extract_dir": "fio-3.18-x86"
+        }
+    },
+    "bin": "fio.exe",
+    "checkver": {
+        "url": "https://bsdio.com/fio/",
+        "regex": "fio-([\\d.]+) released"
+    },
```
Comment (Ash258) [inline bucket/fio.json]:
```suggestion
    "checkver": "fio-([\\d.]+) released",
```

**Comment by Ash258** [comment]
Why do you request review??

image

...

**Comment by AntonOks** [comment]
> 
> 
> Why do you request review??
> 
> image
> 
> ...

just to make sure there is no misunderstanding... you want me to place it in https://github.com/ScoopInstaller/Main instead here? right?

**Pull request closed without merging**